domains/tcm.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheCatsMo",
+        "email": "real.p.e.f.a@gmail.com"
+    },
+    "record": {
+        "CNAME": "themooingcat.netlify.app"
```

Comment:
Netlify is not suppored

Reply:
Why?

Reply:
Also, do you have any recommendations as to what host I can use that is supported?

Reply:
> Why?

It requires a subdomain of the is-a.dev domain, and if you verify yours, no one else would be able to verify theirs
that's why no one can use netlify (same goes for vercel)

Reply:
> Also, do you have any recommendations as to what host I can use that is supported?

if your site is static you can try github pages or cloudflare pages
https://docs.is-a.dev has the guides for some services